features/support/hooks.js

```diff
@@ -1,6 +1,6 @@
 import mongoose from 'mongoose';
 import Election from '../../app/models/election';
-import ElectionTypes from '../../app/models/utils';
+import { ElectionSystems as ElectionTypes } from '../../app/types/types';
```

Comment:
Yeah this was just lazyness :) It was quicker to just change this one line in all files rather than change all occurrences of the variable. IMO it doesn't matter since it makes sense in any case, although I do agree it's marginally better to use the same name everywhere :shrug: